net-libs/gnutls/gnutls-3.4.10.recipe

```diff
@@ -75,8 +88,14 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	runConfigure ./configure --without-p11-kit --disable-doc --disable-nls \
-		--enable-openssl-compatibility
+	if [ -n "$secondaryArchSuffix" ]
+	then
+		runConfigure ./configure --without-p11-kit --disable-nls \
+			--enable-openssl-compatibility --disable-doc
+	else
+		runConfigure ./configure --without-p11-kit --disable-nls \
+			--enable-openssl-compatibility
+	fi
```

Comment:
You could define a variable with "--disable-doc" for secondary Arch, and use this variable with configure.
This way you don't have two copies of runConfigure.